app/src/main/java/comp3350/mealbuddy/objects/CalorieGoal.java

```diff
@@ -0,0 +1,17 @@
+package comp3350.mealbuddy.objects;
+
+import androidx.annotation.Nullable;
+
+public class CalorieGoal extends Goal {
+
+    public CalorieGoal(int lowerBound, int upperBound, GoalType goalType) {
```

Comment:
Did you add the Calorie goal in the GoalType_t?

Reply:
tf is GoalType_t.  Isnt it just ratio or quantity

Reply:
Oh you right, forgot what that was lul